client/trino-client/src/main/java/io/trino/client/StatementClientV1.java

```diff
@@ -101,7 +101,7 @@ public StatementClientV1(OkHttpClient httpClient, ClientSession session, String
         this.timeZone = session.getTimeZone();
         this.query = query;
         this.requestTimeoutNanos = session.getClientRequestTimeout();
-        this.user = session.getUser();
+        this.user = session.getPrincipal();
```

Comment:
perhaps rename the field to `principal`.

Reply:
I think `user` is right here. I moved the assignment here to be a little clearer, but the value is the provided user, or else the principal as a default.

Reply:
with the fixup it looks good to me now